filebeat/module/postgresql/log/_meta/fields.yml

```diff
@@ -28,28 +43,72 @@
       description: >
         Name given to a query when using extended query protocol. If it is "<unnamed>", or not present,
         this field is ignored.
+    - name: command_tag
+      example: "SELECT"
+      description: >
+        Type of session's current command.
+        The complete list can be found at: src/include/tcop/cmdtaglist.h
+    - name: session_start_time
+      type: date
+      description: Time when this session started.
+    - name: virtual_transaction_id
+      description: Backend local transaction id .
+    - name: transaction_id
+      type: long
+      description: The id of current transaction.
+    - name: sql_state_code
+      description: >
+        State code returned by Postgres (if any).
+        This code is not a number.
+        See also https://www.postgresql.org/docs/current/errcodes-appendix.html
+    - name: detail
+    - name: hint
+    - name: internal_query
+      description: internal query that led to the error (if any).
+    - name: internal_query_pos
+      type: long
+      description: character count of the error position (if any).
+    - name: context
+      description: Error context.
+    - name: query_pos
+      type: long
+      description: character count of the error position (if any).
+    - name: location
+      description: Location of the error in the PostgreSQL source code (if log_error_verbosity is set to verbose).
+    - name: application_name
+      description: Name of the application of this event. It is defined by the client.
+    - name: backend_type
+      example: "client backend"
+      description: >
+        Type of backend of this event.
+        Possible types are autovacuum launcher, autovacuum worker, logical replication launcher,
+        logical replication worker, parallel worker, background writer, client backend, checkpointer,
+        startup, walreceiver, walsender and walwriter.
+        In addition, background workers registered by extensions may have additional types.
 
     - name: error.code
       type: long
```

Comment:
Change to `keyword` in the meantime to avoid errors?

Or stop collecting this and make it an alias to `sql_state_code`?